docs/docs/static-query.md

```diff
@@ -1,10 +1,14 @@
 ---
-title: "Querying data in components using StaticQuery"
+title: Querying data in components using StaticQuery
 ---
 
 Gatsby v2 introduces `StaticQuery`, a new API that allows components to retrieve data via GraphQL query.
 
-## Basic example
+In this guide, we'll walk through an example using `StaticQuery`, and discuss [the difference between a StaticQuery and a page query](/static-query/#how-staticquery-differs-from-page-query).
```

Comment:
This is going to end up with a merge conflict in #7249, but otherwise I'm into it. 😄